spring-boot/admin/src/main/java/eu/coatrack/admin/service/GatewayHealthMonitorService.java

```diff
@@ -91,14 +101,19 @@ public ProxyHealthStatus calculateGatewayHealthStatusSummary(List<HealthDataForO
         return ProxyHealthStatus.NEVER_CONNECTED;
     }
 
-    public List<HealthDataForOneGateway> getGatewayHealthMonitorData() {
+    public HealthDataForOneGatewayPlusSummaryStatus getGatewayHealthMonitorData() {
         List<HealthDataForOneGateway> gatewayDataListForGatewayHealthMonitor = proxyRepository.findAvailable()
                 .stream()
                 .map(proxy -> new HealthDataForOneGateway(proxy))
                 .collect(Collectors.toList());
-
         Collections.sort(gatewayDataListForGatewayHealthMonitor, new GatewayHealthDataComparator());
-        return gatewayDataListForGatewayHealthMonitor;
+        ProxyHealthStatus gatewayHealthStatusSummary = calculateGatewayHealthStatusSummary
+                (gatewayDataListForGatewayHealthMonitor);
+
+        HealthDataForOneGatewayPlusSummaryStatus healthDataForOneGatewayPlusSummaryStatus =
```

Comment:
This local variable is not required, as it is directly returned, please just use ` return new ...`

Reply:
I agree and it is Done